supervisor/store/git.py

```diff
@@ -127,6 +127,8 @@ async def clone(self) -> None:
                     suggestions=[
                         SuggestionType.EXECUTE_RELOAD
                         if self.builtin
+                        or self.path.stem
+                        in (addon.repository for addon in self.sys_addons.installed)
                         else SuggestionType.EXECUTE_REMOVE
```

Comment:
I put this PR up because I had completed the work. But after talking it over with others in discord some more I'm not sure we should ever suggest removal here, even if there are no addons installed. Transient github issues seem to put us on this error path and that's not a good reason for removing something the user configured. 

Looking to learn more about why we currently suggest removal here to make sure we handle the intended use cases and just improve our handling of the unintentional ones.